.git2gus/config.json

```diff
@@ -25,11 +25,11 @@
     "area:xml": "a1aB0000000BPyKIAW"
   },
   "issueTypeLabels": {
-    "type:feature": "",
+    "feature": "USER STORY",
     "type:bug": "BUG P3",
     "type:security": "BUG P2",
     "type:feedback": "",
-    "type:duplicate": "",
+    "status:duplicate": "",
```

Comment:
@daphne-sfdc what does this change affect?

Reply:
@peternhale I believe the person who originally created this config.json file was thinking that these 5 custom labels was used to categorize issues.  So an issue belongs in one of these categories - it's either a feature request, bug, security issue, general feedback for us, duplicate of another issue, or community contribution.  Thus, there are labels listed under `issueTypeLabels` that are not mapped to anything because we don't want to create GUS WIs for issues with those types of labels.

As for why I made that change, it's because the tag for duplicate issues is called `status:duplicate`, not `type:duplicate`.  I guess the tag name got changed between when the tag was first created and today.  I didn't get rid of Line 32 because of my assumption of the above, but really Line 32 does nothing.

What do you think - should I get rid of Line 32?  If yes, then the same applies to Line 31 because the `type:feedback` tag isn't mapped to anything either.